src/components/form/controls.tsx

```diff
@@ -36,7 +36,6 @@ export const Label = styled.label<{ required?: boolean }>`
   font-weight: bold;
   font-size: 12px;
   line-height: 110%;
-  color: #668cff;
```

Comment:
Thanks for the comment. On the legacy career form the black colors will not fit. 
As soon as the new design (with white background) is also implemented on the application page, it will fit again. I suggest keeping the black font from this PR because the new application page design will be implemented sooner or later. What do you think @georgiee?

<img width="886" alt="Bildschirmfoto 2021-12-07 um 15 40 43" src="https://user-images.githubusercontent.com/57712895/145049611-ab52b760-3fa0-4dc7-946d-6211148a5944.png">